CONTRIBUTING.md

```diff
@@ -0,0 +1,162 @@
+[_good first issue_](https://github.com/rpm-software-management/dnf5/labels/good%20first%20issue) label.
+These issues are suitable for contributors who are not core maintainers and want to get familiar with the codebase.
+
+Note that sometimes there can be no issues with the _good first issue_ label. Do not worry! There is likely still
+something for you to work on. If you want to contribute but you do not know where to start or cannot find a suitable
+issue, ask for help by reaching out to us on one of our [channels](#asking-for-help).
+
+If there is an issue that you would like to work on, add a comment saying
+that you want to work on this issue, for example, "I want to work on this", and assign the issue to yourself.
+
+## Asking for help
+
+If you need any help when contributing to the project, reach out to us either of the following ways:
+
+* Leave a comment in the original github issue.
+* Send a message to our mailing list: [sst-cs-software-management@redhat.com](mailto:sst-cs-software-management@redhat.com)
+* Ask us on our IRC channel: [#dnf](irc:/chat.libera.org/#dnf)
+
+## Submitting a contribution
+
+Once you have a contribution ready, complete the following steps to submit the contribution:
+
+1. Submit a pull request (PR).
+   1. Set the necessary [labels](https://github.com/rpm-software-management/dnf5/labels).
+      * If you need help while working on the PR, set the
+        [_work in progress_](https://github.com/rpm-software-management/dnf5/labels/work%20in%20progress) label
+        and wait for the PR to be triaged. Triaging usually happens on Mondays or Thursdays.
+      * If you are ready for a code review by one of the maintainers, set the
+        [_ready for review_](https://github.com/rpm-software-management/dnf5/labels/ready%20for%20review) label.
+2. Review the PR.
+   1. Discuss the PR with one of the maintainers. Note that the maintainers might request some changes.
+   2. Review the PR again.
+3. Merge the PR.
+
+## Setting up a development environment
+
+To setup a development environment, complete the following steps:
+
+1. Install build requirements:
+
+   ```
+   dnf builddep dnf5.spec #[--define '_without_<option> 1 ...]
+   ```
+
+2. Build DNF5:
+
+   ```
+   mkdir build
+   cd build
+   cmake ..
+   # or cmake .. [-DWITH_<OPTION>=<ON|OFF> ...]
+   make
+   ```
+
+3. Run the unit-tests. To run the tests, follow the steps to build the code and then run this code:
+
+   ```
+   # from the 'build' directory
+   CTEST_OUTPUT_ON_FAILURE=1 make test
+   ```
+
+   Alternatively, you can execute tests in a verbose mode:
+
+   ```
+   # from the 'build' directory
+   make test ARGS='-V'
+   ```
+
+4. Install DNF5. To install the built package, we recommend to build an rpm package.
+
+   You can build the rpm package either of the following ways:
+
+   - Build the rpms by using tito:
+
+     ```
+     tito build --rpm --test
+     ```
+
+   - Build the rpms from git:
```

Comment:
```suggestion
   - Build the rpm by using tito:

     ```
     tito build --rpm --test
     ```

   - Build the rpm from git:
```

Reply:
it's actually more than one rpm being built